### DISCUSSION TITLE
Make table cells clickable

### DISCUSSION THREAD

**Discussion by dannyshin76** [Budibase/budibase#5946] [Ideas]
I've been extensively using Budibase to create several apps for months. From my experience and actual user feedback, something like below would make the table UI so much more streamlined and intuitive:

![image](https://user-images.githubusercontent.com/93232582/168960851-1b8fc448-b286-4bf3-8a17-e2f1a186bb86.png)
1. Available actions are the same as regular buttons
2. The clickable should be the text inside the cell. So an empty cell can't be clicked.
3. Table row data should be accessible via handlebar and JS (the same as when we add buttons inside the table)

I know similar improvement has been requested several times, so it may be already on your roadmap :)

**Comment by melohagan** [comment]
Hi @dannyshin76 

It's a nice idea, and thank you for the design.

It has been raised as part of a larger table format improvements piece: https://github.com/Budibase/budibase/issues/3273
